### ISSUE TITLE
Link to elasticsearch datatypes page

### ISSUE THREAD

**Issue by shashigharti** [frictionlessdata/website#299]
The link to "Elasticsearch types" in data types page(https://specs.frictionlessdata.io/table-schema/#title)
takes to elastic search home page "elastic.co". 

Is it related to this page "https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-types.html" instead?
![elasticsearch](https://user-images.githubusercontent.com/5582809/162428318-ccc2644a-a811-454d-91b9-b0d9f63f2e2c.png)

**Comment by lwinfree** [comment]
Yes @shashigharti I think that looks like a better link! I am going to transfer this issue to the specs repo, where the code lives for the specs site. Then, can you open a PR to change this link? (Note: this is low priority, so you can do it whenever you have time). Good catch!
